### PR TITLE
Ks 2022 04 add statement to comment serializer

### PR DESCRIPTION
**Pull request by Rineee**
https://github.com/liqd/adhocracy4/pull/1050 should be merged first and then the A4 hash updated here..

### CODE REVIEW

**Comment by khamui** [comment]
working fine! I merged the corresponding a4 and updated this to use the latest main. Will test this further once its merged and connected to frontend components @Rineee .

File: apps/moderatorfeedback/serializers.py [hunk 1]
```diff
@@ -26,3 +28,36 @@ def get_last_edit(self, statement):
...
+        model = Comment
+        read_only_fields = \
+            a4_serializers.CommentSerializer.Meta.read_only_fields + \
+            ('moderator_comment_statement',)
+        exclude = ('creator',)
+
+
+class CommentWithStatementListSerializer(CommentWithStatementSerializer):
+    """Serializer for the comments to be used when viewed as list."""
+
```
Comment (fuzzylogic2000) [inline apps/moderatorfeedback/serializers.py:47]: Does it work that empty? No `pass` or anything like that?

Comment (Rineee) [reply apps/moderatorfeedback/serializers.py:47]: I was also wondering, just copied that as it is from A4 and it seems to work, yes :woman_shrugging: 
https://github.com/liqd/adhocracy4/blob/84d56f6479d7072f15a4b06011d87f41027e8ca1/adhocracy4/comments_async/serializers.py#L202

Comment (fuzzylogic2000) [reply apps/moderatorfeedback/serializers.py:47]: Well, then. Ehm. Let's keep it like that? Or try if it still works when we do add the `pass`? Somehow looking at it makes me feel slightly uneasy. :rofl:

**Pull request merged**